docs/content/preview/drivers-orms/python/postgres-psycopg2.md

```diff
@@ -202,6 +202,10 @@ Query returned: John, 35, Python
 
 If there is no output or you get an error, verify the parameters included in the connection string.
 
+### Limitation
```

Comment:
I think we usually call these sections "Limitations", even if there's only one listed

Reply:
Thanks thats good to know.